frontend/src/app/testResults/TestResultsList.tsx

```diff
@@ -411,15 +411,20 @@ export const DetachedTestResultsList = ({
                 onClick={() => {
                   setDebounced("");
                   clearFilterParams();
-                  (document.querySelector(
-                    "input[id=start-date]"
+                  (document.getElementById(
```

Comment:
I didn't know about `getElementById` originally and since that's exactly what we are doing I thought it would be more clear to do that rather than use the `querySelector`.

Reply:
Makes sense to me! ty 😄